jumpscale/sals/reservation_chatflow/__init__.py

```diff
@@ -26,7 +26,7 @@ def wait_http_test(url: str, timeout: int = 60, verify: bool = True) -> bool:
     start = time()
     while time() - start < timeout:
         try:
-            if check_url_reachable(url, timeout=1, verify=verify):
+            if check_url_reachable(url, timeout=1, verify=verify, status_code=status_code):
```

Comment:
can we use check_url_reachable from nettools sal here instead of implementing the function ?

Reply:
nettools wasn't merged by that time, this one is used any many times over reservation chatflow / sals, this will need a full refactor